docs/app/Examples/collections/Grid/Variations/GridTextAlignmentJustifiedExample.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Grid, Menu } from 'stardust'
+
+// TODO: Update <Menu> usage after v1 API
+
+const { Column, Row } = Grid
+
+const GridTextAlignmentJustifiedExample = () => (
+  <Grid textAlign='center'>
```

Comment:
What I mean is that the filename and the const name are called "Text Alignment Justified" but the prop is set to `textAlign='center'`.  These two are conflicting.  I would think the filename and const name should be "Text Align Center" or the prop should be `textAlign='justified'` so they are matching.